adr/ADR-40.md

```diff
@@ -2,14 +2,14 @@
 
 |Metadata|Value|
 |--------|-----|
-|Date    |2023-10-12 |
-|Author  |@Jarema |
-|Status  |`Implemented`|
-|Tags    |client, server|
+| Date    | 2023-10-12    |
+| Author  | @Jarema       |
+| Status  | Implemented   |
+| Tags    | spec          |
```

Comment:
I'd keep the `client, server` here, so it shows up in those sections also for people to find.  It's not just a spec, its related to multiple things?

Reply:
Think you should also run `go run main.go > README.md`